packaging/faucet/miden-faucet.service

```diff
@@ -7,8 +7,8 @@ WantedBy=multi-user.target
 
 [Service]
 Type=exec
-Environment="RUST_LOG=info"
-ExecStart=/usr/bin/miden-faucet start --config /etc/opt/miden-faucet/miden-faucet.toml
+EnvironmentFile=/lib/systemd/system/miden-faucet.env
```

Comment:
@Mirko-von-Leipzig comment https://github.com/0xMiden/miden-node/pull/976#discussion_r2151502664

---

Should be as simple as editing `/lib/systemd/system/miden-faucet.env` and restarting service. No need for override file. Would expect to use an override file to change things specific to systemd rather than the faucet app e.g. these settings

```
RestartSec=5
Restart=always
```